soft4pes/model/machine/induction_machine.py

```diff
@@ -60,18 +61,17 @@ class InductionMachine:
         Determinant.
     kT : float
```

Comment:
BTW this kT is tricky: It can be confused for kTs. I am wondering if we should denote it differently. Strictly speaking, it should be fine

Reply:
kT will be soon in a class inductionMachineParameters and used with par.kT (or similar), so it cannot be mixed